src/debug/netcoredbg/manageddebugger.cpp

```diff
@@ -31,7 +31,6 @@ using std::map;
 #include <pthread.h>
 #include <link.h>
 #include <sys/types.h>
-#include <sys/wait.h>
```

Comment:
Please note, we already have this code refactored in development repo for fix Linux build and fix MacOS build issue related to recursive mutex (hope, I sync it during this week with github.com).

Reply:
Thanks for the update! I was just working on the macOS build fix myself but in that I guess I'll wait until you have it ready. :smiley: